package.json

```
@@ -19,6 +19,7 @@
     "scripts:analyzeMessages": "lerna run analyzeMessages --scope='scripts'",
     "scripts:scrubMessages": "lerna run scrubMessages --scope='scripts'",
     "scripts:findFaq": "lerna run findFaq --scope='scripts'",
+    "scripts:getConversationText": "lerna run --scope='scripts' getConversationText -- ",
```

```suggestion
    "scripts:getConversationText": "lerna run --scope='scripts' getConversationText",
```

q: not sure if it'd work adding the `--` here

---

Yeah I think it needs the `--` in the script definition. Without it both of these calls fail:

```
npm run scripts:getConversationText 65aaf4dd36a5cac6e828b6be
```

```
npm run scripts:getConversationText -- 65aaf4dd36a5cac6e828b6be
```

with this error

```
ERR! lerna Unknown argument: 65aaf4dd36a5cac6e828b6be
```